library/proc_macro/src/bridge/server.rs

```diff
@@ -8,34 +8,25 @@ use super::client::HandleStore;
 /// Declare an associated item of one of the traits below, optionally
 /// adjusting it (i.e., adding bounds to types and default bodies to methods).
 macro_rules! associated_item {
-    (type FreeFunctions) =>
-        (type FreeFunctions: 'static;);
-    (type TokenStream) =>
-        (type TokenStream: 'static + Clone;);
-    (type TokenStreamBuilder) =>
-        (type TokenStreamBuilder: 'static;);
-    (type TokenStreamIter) =>
-        (type TokenStreamIter: 'static + Clone;);
-    (type Group) =>
-        (type Group: 'static + Clone;);
-    (type Punct) =>
-        (type Punct: 'static + Copy + Eq + Hash;);
-    (type Ident) =>
-        (type Ident: 'static + Copy + Eq + Hash;);
-    (type Literal) =>
-        (type Literal: 'static + Clone;);
-    (type SourceFile) =>
-        (type SourceFile: 'static + Clone;);
-    (type MultiSpan) =>
-        (type MultiSpan: 'static;);
-    (type Diagnostic) =>
-        (type Diagnostic: 'static;);
-    (type Span) =>
-        (type Span: 'static + Copy + Eq + Hash;);
+    (type FreeFunctions) => (type FreeFunctions: 'static;);
+    (type TokenStream) => (type TokenStream: 'static + Clone;);
+    (type TokenStreamBuilder) => (type TokenStreamBuilder: 'static;);
+    (type TokenStreamIter) => (type TokenStreamIter: 'static + Clone;);
+    (type Group) => (type Group: 'static + Clone;);
+    (type Punct) => (type Punct: 'static + Copy + Eq + Hash;);
+    (type Ident) => (type Ident: 'static + Copy + Eq + Hash;);
+    (type Literal) => (type Literal: 'static + Clone;);
+    (type SourceFile) => (type SourceFile: 'static + Clone;);
+    (type MultiSpan) => (type MultiSpan: 'static;);
+    (type Diagnostic) => (type Diagnostic: 'static;);
+    (type Span) => (type Span: 'static + Copy + Eq + Hash;);
```

Comment:
The formatting was pretty intentional but I suppose it's still readable with your flattening. Ideally this could just map to the bounds, but since we can't use macro invocations in bounds it's not really possible.

Reply:
I find the original style to be very difficult to read. I understand that it makes the `(type` be aligned vertically, but that kind of line alternation makes my eyes unhappy.